### PR TITLE
feat(*): allow quote indent props styling - I125

### PR DESCRIPTION
**Pull request by mbtamuli**
# Issue #125
Allow for props to dictate the indentation of the block quotes.

### Changes
- Added `INDENT` style prop as part of the `BLOCKQUOTE` prop.

Without indent:
auto

With indent of `15px` defined by prop:
15px

Signed-off-by: Mriyam Tamuli <mbtamuli@gmail.com>

### CODE REVIEW

**Comment by irmerk** [comment]
Flagging for @michael-grover

**Comment by Michael-Grover** [comment]
lgtm

**Comment by mbtamuli** [comment]
> I think we could keep consistent with QUOTE_COLOR and make this QUOTE_INDENT to be a bit more clear.
@irmerk Made the changes requested. Also added this to the README

File: README.md [hunk 1]
```diff
@@ -122,9 +122,10 @@ This is an object with the following possible css inputs as strings:
   - `FONT_FAMILY`
   - `FONT_COLOR`
   - `FONT_STYLE`
+  - `FONT_SIZE`
```
Comment (irmerk) [inline README.md]: Oopsie, I must have missed that in the past.

**Pull request merged**